browser-test/src/admin/admin_predicates.test.ts

```diff
@@ -399,6 +399,65 @@ test.describe('create and edit predicates', () => {
     })
   })
 
+  test('eligibility message field is available to use', async ({
+    page,
+    adminQuestions,
+    adminPrograms,
+    adminPredicates,
+  }) => {
+    await loginAsAdmin(page)
+    await enableFeatureFlag(page, 'customized_eligibility_message_enabled')
+    const programName = 'Test eligibility message field availbale to use'
+    const firstScreen = 'Screen 1'
+    const secondScreen = 'Screen 2'
+
+    await test.step('Adds a program with two screens', async () => {
+      await adminQuestions.addTextQuestion({
+        questionName: 'show-predicate-q',
+        description: 'desc',
+        questionText: 'text question',
+      })
+      await adminQuestions.addTextQuestion({
+        questionName: 'show-other-q',
+        description: 'desc',
+        questionText: 'conditional question',
+      })
+
+      await adminPrograms.addProgram(programName)
+      await adminPrograms.editProgramBlockUsingSpec(programName, {
+        name: firstScreen,
+        description: 'first screen',
+        questions: [{name: 'show-predicate-q'}],
+      })
+      await adminPrograms.addProgramBlockUsingSpec(programName, {
+        name: secondScreen,
+        description: 'screen with predicate',
+        questions: [{name: 'show-other-q'}],
+      })
+    })
+
+    await test.step('Eligibility message field is visible', async () => {
+      await adminPrograms.goToEditBlockEligibilityPredicatePage(
+        programName,
+        firstScreen,
+      )
+      await validateScreenshot(page, 'edit-predicate-eligibility-msg-enabled')
```

Comment:
**suggestion (blocking):** We're trying to avoid having very many screenshots.  I don't think this one is needed since we have the other screenshot below on line 457.

Reply:
Makes sense! Deleted this screenshot and only keeping the one on line 457.